test/example.cks.html

```diff
@@ -59,6 +59,7 @@
           data-typeaheadtype="local"
           data-typeaheadsource="topics"
           data-evidence="cks"
+          data-skip-link-id="test-id"
```

Comment:
We're using `data-skip-link-id="test-id"` but there's no `test-id` div

Reply:
Your're right it would be better to have main-content div to make intentions clearer. I will add this in and change test-d to main-content. Regardless the tests will pass as I am testing that the skip to next link will take you to a custom id if supplied (by checking the url for the id).